src/validators/base.js

```diff
@@ -67,7 +67,7 @@ class BaseValidator {
 				if (action[paramName] && typeof action[paramName] === "object") {
 					const check = self.compile(action[paramName]);
 					return function validateContextParams(ctx) {
-						let res = check(ctx.params != null ? ctx.params : {});
+						let res = check(ctx.params != null ? ctx.params : {}, { meta: ctx });
```

Comment:
I'm thinking on that it's enough that we use the whole `meta` to point to the context. Because if the future we want to add more "things" to the meta, we can't, or it can cause breaking change. So maybe it can `{meta: { ctx } }`.

Reply:
Are we sure we want to do so much nesting of objects with 1 property?
If we want to add something else in the future, then we can do:
```js
{ meta: { ...ctx, ...etc }}
```

Reply:
ok.